exampleSite/go.mod

```diff
@@ -0,0 +1,5 @@
+module github.com/halogenica/beautifulhugo/exampleSite
+
+go 1.20
+
+require github.com/halogenica/beautifulhugo v0.0.0-20230624134853-c9856df47194 // indirect
```

Comment:
Do these have to be updated regularly?

Reply:
> Do these have to be updated regularly?

To avoid this, you can write

```
require github.com/halogenica/beautifulhugo latest
```

I discarded the idea to make the example site a second, separate module, I think it's better to stick with the current form.